uniffi_aries_vcx/demo/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">demo</string>
-</resources>
\ No newline at end of file
+    <string name="serviceEndpoint">https://b199-27-57-116-96.ngrok-free.app</string>
```

Comment:
Does this need to be set as an env var? Or any other secure and flexible way?

Reply:
to be honest, i'd just make a `Constants.kt` file, and put this value in as a `BASE_RELAY_ENDPOINT` const val, and add a comment above it explaining what it means